src/Extensions/AzureBlobStorage/AzureBlobStorage.csproj

```
@@ -2,27 +2,15 @@
 
   <PropertyGroup>
     <TargetFrameworks>$(ToolTargetFrameworks)</TargetFrameworks>
-    <OutputType>Exe</OutputType>
-    <RootNamespace>Microsoft.Diagnostics.Tools.Monitor.Egress.AzureBlob</RootNamespace>
+    <RootNamespace>Microsoft.Diagnostics.Monitoring.AzureStorage</RootNamespace>
```

```suggestion
    <RootNamespace>Microsoft.Diagnostics.Monitoring.AzureBlobStorage</RootNamespace>
```

The primary purpose of the assembly is to store artifacts into Azure Blobs and naming it this makes it clear that this extension is for that purpose. The namespace should follow in the same vein.